kubernetes/samples/charts/util/setup.sh

```diff
@@ -1,12 +1,36 @@
 #!/bin/bash
-# Copyright (c) 2018, 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+# Copyright (c) 2018,2019,2020 Oracle Corporation and/or its affiliates.  All rights reserved.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
 
 #  This script is to create or delete Ingress controllers. We support two ingress controllers: traefik and voyager.
 
 MYDIR="$(dirname "$(readlink -f "$0")")"
 VNAME=voyager-operator  # release name of Voyager
 TNAME=traefik-operator  # release name of Traefik
+VSPACE=voyager # NameSpace for Voyager
+TSPACE=traefik   # NameSpace for Traefik
+
+helm version --short | grep v3
+[[ $? == 0 ]] && HELM_VERSION=V3
+[[ $? == 1 ]] && HELM_VERSION=V2
+
+echo "Detected Helm Version [$HELM_VERSION]"
+
+if [ "$HELM_VERSION" == "V3" ]; then
```

Comment:
Reverse the logic here and check for V2 instead of V3 (more future proofing).

Reply:
Yes, I reversed the logic in recent commit